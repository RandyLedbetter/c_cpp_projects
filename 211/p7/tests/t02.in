insert Chico Sacramento 90
lookup Chico
lookup Sacramento
