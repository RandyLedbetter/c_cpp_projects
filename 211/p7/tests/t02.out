Chico: none, Sacramento(90), none
Sacramento: Chico(90), none, none
